packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -163,11 +163,34 @@ function normalizeConfig (config, serviceIdentifier) {
       break
   }
 
-  return Object.assign({}, config, specificConfig, {
+  const baseAWSBatchPropagationValue = process.env.DD_TRACE_AWS_SDK_BATCH_PROPAGATION_ENABLED
+  const baseAWSBatchPropagationEnabled = baseAWSBatchPropagationValue ? isTrue(baseAWSBatchPropagationValue) : false
+
+  // check if AWS batch propagation or AWS_[SERVICE] batch propagation is enabled via env variable
+  const serviceId = serviceIdentifier.toUpperCase()
```

Comment:
Do the service identifiers consist entirely of a-z characters? Are there ever hyphens?

Reply:
I'm not 100% sure, but we already use this pattern within line 109 of this file and it hasn't caused any problems